cwd/editor.js

```diff
@@ -1,3 +1,60 @@
+const update = (btn) => {
+  const viewNum = btn.id.substring(8, 9);
+  const gaugeNum = btn.id.substring(10, 11);
+  const msgNum = btn.id.substring(12, );
+
+  const prob = [];
+  for (const x of Array(5).keys()) {
+    prob.push($('#prob-' + (btn.id.substring(8, )) + "-" + (x + 1)).val());
+  }
+
+  const settings = {
+    "url": views[viewNum - 1] + "gauges/" + gaugeNum + "/messages/" + msgNum,
+    "type": "POST",
+    "timeout": 0,
+    "headers": {
+      "Content-Type": "application/x-www-form-urlencoded"
+    },
+    "data": {
+      "message": "\"" + $('#input-' + (btn.id.substring(8, ))).val() + "\"",
+      "probability": `\[${prob}\]`,
+      "pass": "\"" + $('#passIn').val() + "\""
+    }
+  };
+
+  $.ajax(settings).done(function(response) {
+    console.log(response);
```

Comment:
I am probably interested in some error handling on the frontend.  For instance, if there is an error, can we at least send it to the client using `alert`?

Reply:
This still is a relevant point, but it doesn't matter.